src/components/WebexAvatar/README.md

```diff
@@ -1,9 +1,22 @@
 # Webex Avatar Component
+Webex avatar component displays the avatar and status of a _Webex Teams_ user.
 
-Webex avatar component displays the avatar of a _Webex Teams_ user with a given `personID` and `adapter` object.
+![Webex Avatar](./WebexAvatar.png)
```

Comment:
it be nice to center this, but not a blocker.

Reply:
I centered but i don't know if I like it, it's a bit small of an image, so it kinda gets lost